src/app/modules/users/components/users-list/users-list.component.ts

```diff
@@ -0,0 +1,55 @@
+import { AfterViewInit, Component, EventEmitter, Input, OnDestroy, OnInit, Output, ViewChild } from '@angular/core';
```

Comment:
EventEmitter, Input, and Output are not used imports

Reply:
I have already removed them, thanks